src/icon-manifest/supportedExtensions.ts

```diff
@@ -91,6 +91,7 @@ export const extensions: IFileCollection = {
     { icon: 'bolt', extensions: [], languages: [languages.bolt], filename: true, format: FileFormat.svg },
     { icon: 'bower', extensions: ['.bowerrc', 'bower.json'], filename: true, format: FileFormat.svg },
     { icon: 'buckbuild', extensions: ['.buckconfig'], filename: true, format: FileFormat.svg },
+    { icon: 'bundler', extensions: ['gemfile', 'gemfile.lock'], format: FileFormat.svg },
```

Comment:
it is already defined a line below?

Reply:
it's defined below with `filename: true` and works only for files `gemfile` and `gemfile.lock`